jekyll/_cci2/configuration-reference.md

```diff
@@ -739,10 +739,70 @@ Key | Required | Type | Description
 version | Y | String | Should currently be `2`
 {: class="table table-striped"}
 
-#### **<`workflow_name`>**
+### **<`workflow_name`>**
 
 A unique name for your workflow.
 
+#### **`triggers`**
+Specifies which triggers will cause this workflow to be executed. Default behavior is to trigger the workflow when pushing to a branch. 
+
+Key | Required | Type | Description
+----|-----------|------|------------
+triggers | N | String | Should currently be `schedule`.
+{: class="table table-striped"}
+
+##### **`schedule`**
+A workflow may have a `schedule` indicating it runs at a certain time, for example a nightly build that runs every day at 12am UTC:
```

Comment:
While it's probably clear from the nested context, perhaps using "trigger", e.g.:

> Trigger a workflow to run at a certain time, for example a nightly build that runs every day at 12am UTC:

Reply:
I'm going to leave this as-is for now